crates/bevy_ui/src/widget/text.rs

```diff
@@ -49,10 +63,16 @@ pub fn text_system(
     mut text_queries: ParamSet<(
         Query<Entity, Or<(Changed<Text>, Changed<Style>)>>,
         Query<Entity, (With<Text>, With<Style>)>,
-        Query<(&Text, &Style, &mut CalculatedSize)>,
+        Query<(&Text, &Style, &mut CalculatedSize, Option<&Parent>)>,
     )>,
+    parent_query: Query<&Style>,
```

Comment:
```suggestion
    style_query: Query<&Style>,
```

Nit: `parent_query` seems misleading here. I get it's used to query the style of the parent, but where it's used, I think it would be more readable if it was called `style_query`.